var/spack/repos/builtin/packages/py-kubernetes/package.py

```diff
@@ -28,7 +32,8 @@ class PyKubernetes(PythonPackage):
     depends_on('py-six@1.9.0:', type=('build', 'run'))
```

Comment:
Looks like Python 3.6+ is required in the latest version, can you add this?

Reply:
Good catch, done!